src/main/java/filter/expression/QualifierType.java

```diff
@@ -50,7 +50,7 @@
         defaultCompletions.removeAll(Arrays.asList("m", "as", "au", "s"));
 
         defaultCompletions.addAll(Arrays.asList("closed", "open", "issue", "pr",
-            "pullrequest", "read", "unread",  "merged", "unmerged", "comments",
+            "pullrequest", "read", "unread", "me", "merged", "unmerged", "comments",
```

Comment:
Does the alias `me` for `assignee` exist? I was using it in my previous comment for simplicity.